centreon/tests/e2e/features/Cloud-notifications/04-notification-edit.feature

```diff
@@ -17,22 +16,17 @@ Feature: Editing Notification Rule configuration
     And the notification refresh delay has been reached
     Then only notifications for status changes of the updated resource parameters are sent
 
-  Scenario Outline: Editing a Notification Rule users configuration
+  Scenario: Editing a Notification Rule users configuration
     When the user selects the edit action on a Notification Rule
-    And the user changes the <user_type> configuration
+    And the user changes the contact configuration
     And the user saves to confirm the changes
     And the notification refresh delay has been reached
-    Then the notifications for status changes are sent only to the updated <user_type>
-
-    Examples:
-      | user_type      |
-      | contact        |
-      | contact groups |
```

Comment:
no need to test with contact groups ?

Reply:
At this moment, Contact Groups is not available, is removed at the end before release 23.10 on the notifications panel.